app/models/comment.rb

```diff
@@ -1,10 +1,10 @@
 class Comment < ActiveRecord::Base
 
-  default_scope { order(created_at: :desc) }
+  default_scope {order(created_at: :desc) }
 
   belongs_to :user
   belongs_to :topic
-  has_many :replies, class_name: 'Comment', foreign_key: 'parent_comment_id'
+  has_many :replies, class_name: "Comment", foreign_key: 'parent_comment_id'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.